docs/user-guide/kubernetes.md

```diff
@@ -562,7 +562,11 @@ could set `disablePassHostHeaders` to true in your toml file and then enable pas
 the host header per ingress if you wanted.
 
 ## Excluding an ingress from Træfik
-You can control which ingress Træfik cares about by using the "kubernetes.io/ingress.class"
+
+You can control which ingress Træfik cares about by using the `kubernetes.io/ingress.class`
 annotation. By default if the annotation is not set at all Træfik will include the
 ingress. If the annotation is set to anything other than traefik or a blank string
 Træfik will ignore it.
+
+
+![](http://i.giphy.com/ujUdrdpX7Ok5W.gif)
```

Comment:
See https://docs.traefik.io/user-guide/swarm/#access-to-your-apps-through-trfk 😉